### ISSUE TITLE
USWDS - Icons: Confirm if legacy attributes still required 

### ISSUE THREAD

**Issue by mejiaj** [uswds/uswds#5316]
Confirm if SVGs still require aria-hidden and focusable="false" (previously for legacy browsers). From this [comment in issue](https://github.com/uswds/uswds/issues/4307#issuecomment-1559410923).

**Comment by gitstart** [comment]
@mejiaj I would like to pick this up

**Comment by ethangardner** [comment]
@chandracarney - would you mind taking a look at this? We get PRs occasionally that want to remove the `role` from the banner's `<img>` tags - See https://github.com/uswds/uswds/pull/6800. 

@annepetersen - I know the `role` and `aria-hidden` tandem was added for a Safari and VoiceOver bug when an `<img>` used an svg. Do you remember where that issue or research was? I'm not sure if the issue has been patched or not.

**Comment by chandracarney** [comment]
Thanks @ethangardner! @jonathanbobel, @iamjolly, and I met to do an initial triage here, and @jonathanbobel will dive in soon as it needs a further investigation.

**Comment by annepetersen** [comment]
As far as I can tell, we haven't confirmed whether this has been patched or not.

The [link up in the original post here](https://github.com/uswds/uswds/issues/4307#issuecomment-1559410923) is probably the most relevant documentation we've got, but that's from a few years back. 

Unfortunately, the [notes from the meeting](https://docs.google.com/document/d/1gpqTSz_vvlSlW6O6k2MAatpWKAtFLNJ38Q4gSvaOzKE/edit?tab=t.0#heading=h.en2474igono9) that prompted the issue creation are too sparse to pull much from (and are from 2023, so things may well have changed since then anyway).

**Comment by jonathanbobel** [comment]
@ethangardner - general notes: 

1. Decorative banner `<img src="*.svg">` icons: drop `role="img"`. The VoiceOver SVG-as-image bug that required it was [confirmed fixed in Safari 14.1.2 / iOS 14.7.1](https://www.scottohara.me/blog/2021/09/03/img-svg-source.html). Keep `alt=""` and `aria-hidden="true"`. Retest with current Safari + VoiceOver (these SVG files still contain a `<title>`).
2. Keep `aria-hidden="true"` on all decorative images and decorative inline SVGs.
3. Keep `role="img"` on informative inline <svg> that should be announced (banner lock), with an accessible name via `<title>` / `aria-labelledby`.
4. Remove `focusable="false"`. That was for IE11 / pre-Chromium Edge. USWDS no longer supports those browsers.

I can work on 1 and 4 as it relates to this issue.

**Comment by ethangardner** [comment]
@jonathanbobel - Thanks! A PR was opened to remove it from the banner https://github.com/uswds/uswds/pull/6800. Take a look at that PR and see if you think it can be merged.
